### ISSUE TITLE
Cannot install Pynwrfc

### ISSUE THREAD

**Issue by Dinesh-S-Kumar** [SAP/PyRFC#189]
Ive been trying to install Pynwrfc but it is throwing a lot of errors.
Trying with Admin rights did not work as well.

<img width="959" alt="errors" src="https://user-images.githubusercontent.com/68798469/88470450-17959600-cf1a-11ea-9915-8d4a683e3d9e.PNG">

**Comment by bsrdjan** [comment]
Did you try `pip install --user pynwrfc` ?

**Comment by Dinesh-S-Kumar** [comment]
Yes, tried it but still the same error.

**Comment by bsrdjan** [comment]
Two other possibilities:

1) Run the CMD/PowerShell as Administrator, or

2) Install the pynwrfc in a `virtualenv`, which is recommended option: https://stackoverflow.com/questions/32167418/python-pip-install-trouble-shooting-permissionerror-winerror-5-access-is

**Comment by Dinesh-S-Kumar** [comment]
Tried both possibilities but still getting the same error.
Did a fresh install from the beginning in a new system and packages were installed successfully.

**Comment by Dinesh-S-Kumar** [comment]
![Capture](https://user-images.githubusercontent.com/68798469/88582755-d4980780-d06c-11ea-9e9a-b36ecc426f6e.PNG)

After full setup, tried to run the below code to test the setup, it is throwing error. did i do something wrong here?

from configparser import ConfigParser
from pyrfc import Connection
# from value import *
def main():
	config = ConfigParser()
	config.read('sapnwrfc.cfg')
	params_connection=config._sections['connection']
	conn = Connection(***connection parameters)
	print(conn)
	result = conn.call('STFC_CONNECTION',REQUTEXT=u'Hello Sap')
	print(result)

if __name__=="__main__":
	main()

**Comment by bsrdjan** [comment]
Please post the syntax correct script and full console/error log, starting with script invocation.

**Comment by Dinesh-S-Kumar** [comment]
Sure, Please find the script screenshot and 
![Capture](https://user-images.githubusercontent.com/68798469/88633137-38085080-d0d2-11ea-819c-159c6aa6cdc7.PNG)

full error log below:

C:\Users\dines\PycharmProjects\Android_handling\venv\Scripts\python.exe C:/Users/dines/PycharmProjects/Android_handling/Simple.py
Traceback (most recent call last):
  File "C:/Users/dines/PycharmProjects/Android_handling/Simple.py", line 14, in <module>
    main()
  File "C:/Users/dines/PycharmProjects/Android_handling/Simple.py", line 10, in main
    result = conn.call('STFC_CONNECTION',REQUTEXT=u'Hello Sap')
AttributeError: 'NoneType' object has no attribute 'call'
None

Process finished with exit code 1

**Comment by bsrdjan** [comment]
The error in console indicates that `conn` object could not be created (therefore NoneType) because of:

`unexpected keyword argument 'saprouter'`

What is the content of `sapnwrfc.cfg` ?

**Comment by Dinesh-S-Kumar** [comment]
The content of sapnwrfc.cfg are as follows:

[connection]
# sap system ip
ashost = *ashost
# sap client id
client = 100
# sap router string, optional, format like /H/x.x.x.x/...
#saprouter = 
# sap system number
sysnr = 00
# sap username
user = *UserID
# sap password
passwd = *Password
# language
lang = EN


[gateway]
# gateway host name
gwhost =
# gateway server name
ghserv =
# name under which the python connector will register itself
program_id = PYRFC_SERVER1
# sap router string, optional
saprouter =

i maintained my system ashost,client,sysnr,user and passwd while running the program.
i did try to comment and try the script again but 'conn' object was still None.

**Comment by bsrdjan** [comment]
Could you please run the script from the console command line:

```shell
C:> python Simple.py
```

and post the copy of the console output, starting with script invocation. Attach also in text format, not a screenshot, so that I can try to reproduce:

1. `Simple.py`
2.  `sapnwrfc.cfg`

**Comment by Dinesh-S-Kumar** [comment]
I dont know why this works but it works now. Steps i followed.

1. Uninstalled `pyrfc`
2. Uninstalled `pynwrfc`
3. Uninstalled Python and installed python again.
4. did `pip install pynwrfc`
5. Added python interpreter to `python.exe` in` C:\  AppData` 

I used to uninstall `pyrfc` and install only `pynwrfc`, but of no use. Tried the same thing again along with python files and it somehow works.

Thanks for the assist 👍
